### PR TITLE
EDU-1399 Adjust usage of tables with paging to manage the paging state externally

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-1399

### CODE REVIEW

File: src/components/pages/search.js [hunk 1]
```diff
@@ -33,9 +33,9 @@ function Search({ PageTemplate }) {
   const [selectedTags, setSelectedTags] = useState([]);
   const [displayedRows, setDisplayedRows] = useState([]);
   const [unselectedTags, setUnselectedTags] = useState([]);
-  const [currentPagination, setCurrentPagination] = useState(1);
   const [searchText, setSearchText] = useState(request.query.query);
   const [sorting, setSorting] = useState({ value: 'relevance', direction: 'desc' });
+  const [currentTablePagination, setCurrentTablePagination] = useState({ current: 1, pageSize: 10, showSizeChanger: true });
```
Comment (adrianaluchian) [inline src/components/pages/search.js:38]: It looked a bit buggy that ant was not always showing the size changer (showing it only with a data set of more than 50 rows), so I enabled it all the time.

File: src/components/redaction/redaction-tags-tab.js [hunk 2]
```diff
@@ -129,8 +129,8 @@ function RedactionTagsTab({ documents, mediaLibraryItems }) {
     setDisplayedTableRows(sortedRows);
   }, [allTableRows, filterText, currentTableSorting, tableSorters]);
 
-  const handleTableChange = ({ current, pageSize }) => {
-    setCurrentPagination([current, pageSize].join());
+  const handleTableChange = newPagination => {
+    setCurrentTablePagination(oldPagination => ({ ...oldPagination, ...newPagination }));
```
Comment (adrianaluchian) [inline src/components/redaction/redaction-tags-tab.js:133]: We talked about persisting paging data in the URL for search only, probably for redaction it is not as necessary, but I streamlined the paging here too, for consistency.

Comment (ahelmberger) [reply src/components/redaction/redaction-tags-tab.js:133]: I'm fond of streamlining, so absolutely no objections :rofl:

**Comment by ahelmberger** [comment]
That was a lot smoother than anticipated, very nice :+1:

File: src/components/redaction/redaction-tags-tab.js (hunk 2 above)
Comment (adrianaluchian) [reply src/components/redaction/redaction-tags-tab.js:133]: ❤️

File: src/components/pages/search.js (hunk 1 above)
Comment (ahelmberger) [reply src/components/pages/search.js:38]: You need to actively look into this in order to find that out :roll_eyes:

**Review by ahelmberger — APPROVED** [review]

**Pull request merged**